doc/wave_calib.rst

```diff
@@ -84,6 +85,47 @@ vary (e.g. grating tilts).  We are likely to implement
 this for echelle observations (e.g. HIRES).
 
 
+Identify
+--------
+
+If you would prefer to manually wavelength calibrate, then
+you can do so with the 'identify' task. To launch this task,
+you need to have successfully traced the slit edges (i.e. a
+MasterEdges file must exist), and generated a MasterArc
+calibration frame. To launch a GUI, use the following command:
+
+    pypeit_identify MasterArc_A_1_01.fits
+
+Instructions on how to use this GUI are available by pressing
+the '?' key while hovering your mouse over the plotting window.
+Once you have completed the manual calibration, you can save
+your solution (press key 's' while hovering over the plotting
+window). The, update your .pypeit file with the following:
+
```

Comment:
Same thing here:

```
...
 window). The, update your .pypeit file with the following: 

.. code-block:: ini

    [calibrations]
      [[wavelengths]]
        method=identify

and a GUI will be launched during the reduction. You can now
...
```

Reply:
Thanks again - I'm not so savvy with `.rst` formatting, but will work on it...